arches_references/management/commands/controlled_lists.py

```diff
@@ -218,60 +249,51 @@ def migrate_concept_nodes_to_reference_datatype(self, graph_id):
+                            models.F("config"),
+                            output_field=models.JSONField(),
                         )
-                        .annotate(
-                            without_default=CombinedExpression(
-                                models.F("config_without_i18n"),
-                                "-",
-                                models.Value(
-                                    "defaultValue", output_field=models.CharField()
-                                ),
-                                output_field=models.JSONField(),
-                            )
-                        )
-                        .annotate(
-                            without_default_and_options=CombinedExpression(
-                                models.F("without_default"),
-                                "-",
-                                models.Value(
-                                    "options", output_field=models.CharField()
-                                ),
-                                output_field=I18n_JSONField(),
-                            )
+                    ).annotate(
+                        config_without_options=CombinedExpression(
+                            models.F("config_without_i18n"),
+                            "-",
+                            models.Value("options", output_field=models.CharField()),
+                            output_field=I18n_JSONField(),
                         )
                     )
                     for cross_record in cross_records:
                         # work around for i18n as_sql method issue detailed here: https://github.com/archesproject/arches/issues/11473
                         cross_record.config = {}
                         cross_record.save()
 
-                        cross_record.config = cross_record.without_default_and_options
+                        # Crosswalk concept version of default values to reference versions
+                        original_default_value = (
+                            cross_record.config_without_options.get(
+                                "defaultValue", None
+                            )
+                        )
+                        if original_default_value:
+                            new_default_value = []
+                            if isinstance(original_default_value, str):
+                                original_default_value = [original_default_value]
+                            for value in original_default_value:
+                                value_rec = Value.objects.get(pk=value)
+                                config = {"controlledList": node.collection_id}
+                                new_value = REFERENCE_FACTORY.transform_value_for_tile(
+                                    value=value_rec.value,
+                                    **config,
+                                )
+                                new_default_value.append(new_value[0])
```

Comment:
I think `transform_value_for_tile` returns the input unchanged if no match, so I would expect `[0]` to fail on a string. Is that right?